Fireworks/Calo/src/FWTauProxyBuilderBase.cc

```diff
@@ -35,166 +36,140 @@
 #include "DataFormats/TauReco/interface/BaseTau.h"
 #include "DataFormats/TrackReco/interface/Track.h"
 
-
 static const std::string kTauApexBeamSpot("Place Apex In BeamSpot");
-FWTauProxyBuilderBase::FWTauProxyBuilderBase():
-   m_minTheta(0),
-   m_maxTheta(0)
-{
-}
+FWTauProxyBuilderBase::FWTauProxyBuilderBase() : m_minTheta(0), m_maxTheta(0) {}
 
-FWTauProxyBuilderBase::~FWTauProxyBuilderBase()
-{
-}
-
-   
-void FWTauProxyBuilderBase::setItem(const FWEventItem* iItem)
-{FWProxyBuilderBase::setItem(iItem);
-      if (iItem) {
-      iItem->getConfig()->assertParam(kTauApexBeamSpot, false);
-      }
+FWTauProxyBuilderBase::~FWTauProxyBuilderBase() {}
 
+void FWTauProxyBuilderBase::setItem(const FWEventItem* iItem) {
+  FWProxyBuilderBase::setItem(iItem);
+  if (iItem) {
+    iItem->getConfig()->assertParam(kTauApexBeamSpot, false);
+  }
 }
-      
-void
-FWTauProxyBuilderBase::buildBaseTau( const reco::BaseTau& iTau, const reco::Jet* iJet, TEveElement* comp, FWViewType::EType type, const FWViewContext* vc)
-{
-   // track
-   addLeadTrack( iTau, comp );
-   addConstituentTracks( iTau, comp );
- 
-   // projected markers
-   if (FWViewType::isProjected(type))
-   {
-      double phi   = iTau.phi();
-      double theta = iTau.theta();
-      double size  = 1;
-
-      bool  barrel = (theta< context().caloTransAngle() || theta > (TMath::Pi() - context().caloTransAngle()));
-      float ecalR  = barrel ? context().caloR1() : context().caloR2();
-      float ecalZ  = barrel ? context().caloZ1() : context().caloZ2();
-  
-      TEveScalableStraightLineSet* marker = new TEveScalableStraightLineSet( "energy" );
-     
-      if( type == FWViewType::kRhoZ )
-      {
-         double r(0);
-         ( theta <  context().caloTransAngle() || M_PI-theta < context().caloTransAngle()) ?
-            r = ecalZ/fabs(cos(theta)) :
-            r = ecalR/sin(theta);
-   
-         fireworks::addRhoZEnergyProjection( this, comp, ecalR, ecalZ, m_minTheta-0.003, m_maxTheta+0.003, phi);
-
-         marker->SetScaleCenter( 0., (phi>0 ? r*fabs(sin(theta)) : -r*fabs(sin(theta))), r*cos(theta) );
-         marker->AddLine(0., (phi>0 ? r*fabs(sin(theta)) : -r*fabs(sin(theta))), r*cos(theta),
-                         0., (phi>0 ? (r+size)*fabs(sin(theta)) : -(r+size)*fabs(sin(theta))), (r+size)*cos(theta) );
 
-      }
-      else
-      {
-         std::pair<double,double> phiRange = fireworks::getPhiRange( m_phis, phi );
-         double min_phi = phiRange.first-M_PI/36/2;
-         double max_phi = phiRange.second+M_PI/36/2;
-         if( fabs(phiRange.first-phiRange.first)<1e-3 ) {
```

Comment:
This is one of the changes that were implemented in the master before #34655: is it still acceptable for 9_4_X?